automation_tools/satellite6/upgrade/tasks.py

```diff
@@ -402,10 +402,13 @@ def sync_capsule_tools_repos_to_upgrade(admin_password=None):
     run(initials + 'content-view publish --name {0} '
         '--organization {1}'.format(cv_name, org))
     # promote cv
-    lc_env_id = str(
+    lc_list = str(
         run(initials + 'lifecycle-environment list '
-            '--organization {0} | grep {1}'.format(org, env_name))).split(
-                '|')[0].strip()
+            '--organization {0} | grep {1}'.format(org, env_name))).split('\n')
+    for lc in lc_list:
```

Comment:
Why not grab the CSV output? You can use the csv module to parse the output and access the attributes by the column name.